site/homepage/content/release-notes.md

```diff
@@ -21,6 +21,8 @@ description = "Information about changes in recent Hono releases. Includes new f
   higher rate of Command Consumer unregister calls.
   On stop, the adapter no longer explicitly unregisters Command Consumers and no longer sends `disconnectedTtdEvent`.
   The sending of `connectedTtdEvent` / `disconnectedTtdEvent` is moved from adapter to Command Router.
+  Command Router sends `disconnectedTtdEvent` in case of command from north bound application to disconnected device,
+  that is marked as connected to adapter that was already stopped.
```

Comment:
I find that a bit hard to understand.
How about something like

> Command Router now sends a `disconnectedTtdEvent` when a command is received that is targetted at a device that got disconnected (and didn't reconnect) when a protocol adapter was stopped.

?

Reply:
I agree, what about your version and one additional sentence:

> Command Router now sends a `disconnectedTtdEvent` when a command is received that is targetted at a device that got disconnected (and didn't reconnect) when a protocol adapter was stopped. This happens only once - during the first command.

Reply:
Good point, I would just use `during the processing of the first command` or `for the first command`.

Reply:
Improved.